Assets/MRTK/StandardAssets/EditorUtilities/OnLoadUtilities.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities.Editor
+{
+    [InitializeOnLoad]
+    public static class OnLoadUtilities
+    {
+        private const string ShaderSentinelFile = "MRTK.Shaders.Sentinel";
+        private const string ShaderImportDestination = "MRTK/Shaders";
+
+        static OnLoadUtilities()
+        {
+            ImportShaderFiles();
+            EnsureShaders();
```

Comment:
Should this just be `EnsureShaders()`? Since that method calls `ImportShaderFiles()` itself, this is potentially calling it twice.

Reply:
And `EnsureShaders()` has a guard with `AssetsContainsShaders()`, in case MRTK is imported via .unitypackage and already in the Assets folder

Reply:
that was a mistake on my part based on some testing i was doing.